packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/AppContext.kt

```diff
@@ -105,6 +108,19 @@ class AppContext(
   val reactContext: ReactApplicationContext?
     get() = reactContextHolder.get()
 
+  /**
+   * Provides access to the event emitter
+   */
+  fun eventEmitter(module: Module): EventEmitter? {
+    val legacyEventEmitter = legacyModule<EventEmitter>() ?: return null
+    return KEventEmitterWrapper(
+      requireNotNull(registry.getModuleHolder(module)) {
+        "Cannot create an event emitter for the module that isn't present in the current module registry."
```

Comment:
What *current* means here? 🤔 
```suggestion
        "Cannot create an event emitter for the module that isn't present in the module registry."
```